src/kernels/jupyter/types.ts

```diff
@@ -154,86 +155,17 @@ export interface IJupyterServerProvider {
     getOrStartServer(options: GetServerOptions): Promise<IJupyterConnection>;
 }
 
-export interface IJupyterServerUri {
```

Comment:
Removed all of these duplicate types, we have them here and the extension.d.ts files.
These types are meant to be public for 3rd party extensions.